jobs/dynatrace-oneagent-windows/templates/pre-start.ps1.erb

```diff
@@ -110,7 +110,14 @@ function downloadAgent() {
 
 		Try {
 			installLog "INFO" "Downloading Dynatrace agent from $cfgDownloadUrl to $installerFile"
-			Invoke-WebRequest $cfgDownloadUrl -Outfile $installerFile -UserAgent $userAgent
+			
+			if($cfgProxy -ne "") {
```

Comment:
Also, shouldn't the installer be told that the agent should use the proxy later on?

Currently, we're just downloading the installer via proxy and that's it.